src/main/java/org/example/lesson4/repository/impl/UserRepositoryImpl.java

```diff
@@ -0,0 +1,92 @@
+package org.example.lesson4.repository.impl;
+
+import org.example.lesson4.exception.DataProcessingException;
+import org.example.lesson4.model.User;
+import org.example.lesson4.repository.UserRepository;
+import org.example.lesson4.util.ConnectionUtil;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.ResultSet;
+import java.util.Optional;
+
+public class UserRepositoryImpl implements UserRepository {
+    private static final int MIN_AFFECTED_ROWS_COUNT = 1;
+    private static final String ID = "id";
+    private static final String EMAIL = "email";
+    private static final String PHONE_NUMBER = "phoneNumber";
+    private static final String PASSWORD = "password";
+    private static final String REPEAT_PASSWORD = "repeatPassword";
+
+    @Override
+    public User save(User user) {
+        String sqlRequest = "INSERT INTO users (email, phoneNumber, password, repeatPassword) VALUES (?, ?, ?, ?)";
```

Comment:
There is no need to store `repeatPassword` in DB. This field is required only on the service layer to check if the user passed a valid password and repeatPassword combination.